mod/03_machines/lib/javascript/wagn.js.coffee

```diff
@@ -150,6 +150,9 @@ $(window).ready ->
   $('body').on 'ajax:success', '.slotter', (event, data, c, d) ->
     unless event.slotSuccessful
       $(this).slotSuccess data
+      # should scroll to top after clicking on new page
+      if $(this).hasClass "card-paging-link"
+        $("body").scrollTop 0
```

Comment:
I like the continuous pager idea, too.  This is specifically for the case where there are (old school) paging links at the bottom of a list.  I suspect we'd like to have both.